src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Handlers/DynamicItemHandler.cs

```diff
@@ -26,48 +26,49 @@ public DynamicItemHandler(UnconfiguredProject project)
             _project = project;
         }
 
-        public string ProjectEvaluationRule
-        {
-            get { return Content.SchemaName; }
-        }
-
-        public void Handle(IComparable version, IProjectChangeDescription projectChange, ContextState state, IProjectDiagnosticOutputService logger)
+        public void Handle(IComparable version, IImmutableDictionary<string, IProjectChangeDescription> projectChanges, ContextState state, IProjectDiagnosticOutputService logger)
```

Comment:
Recommend reviewing this file with whitespace turned off, otherwise GitHub overplays the changes.